pkg/client/clientcmd/api/types.go

```diff
@@ -83,6 +83,8 @@ type Context struct {
 	AuthInfo string `json:"user"`
 	// Namespace is the default namespace to use on unspecified requests
 	Namespace string `json:"namespace,omitempty"`
+	// NamespacePath is the path to a kubernetes ns file (~/.kubernetes_ns)
```

Comment:
I think we should remove it when we remove kubecfg.  I thought they used the same file. 

Sent from my iPhone

> On Jan 20, 2015, at 3:57 PM, Jeff Lowdermilk <notifications@github.com> wrote:
> 
> In pkg/client/clientcmd/api/types.go:
> 
> > @@ -83,6 +83,8 @@ type Context struct {
> >  	AuthInfo string `json:"user"`
> >  	// Namespace is the default namespace to use on unspecified requests
> >  	Namespace string `json:"namespace,omitempty"`
> > +	// NamespacePath is the path to a kubernetes ns file (~/.kubernetes_ns)
> +1 to nuking namespace-path. I don't think we have anything using it now, but removing it in a separate PR sgtm.
> 
> —
> Reply to this email directly or view it on GitHub.
>